example/main/default/triggers/CaseCreatedTrigger.trigger

```diff
@@ -1,10 +1,9 @@
-trigger CaseCreatedTrigger on Case(after insert, after update) {
-    System.debug('Running Case Created Trigger.  Update: ' + Trigger.isUpdate + ' Records: ' + Trigger.size);
+trigger CaseCreatedTrigger on Case(after insert) {
     TB_Outbox_Message__c[] messages = new List<TB_Outbox_Message__c>{};
     for (Case c : Trigger.new) {
         messages.add(
             new TB_Outbox_Message__c(
-                Type__c = Trigger.isUpdate ? 'case_updated' : 'case_created',
+                Type__c = 'case_created',
```

Comment:
I ran into recursion after hooking in the "AfterRelayAction" because it updates the case, triggering an `case_updated` event... which then updates the case lol 😬 .

Would be very easy to burn through all you ORG async job limits if this happens.

We could add some static flag to prevent the `Outbox_Message__c` trigger from running if it's in a recursive loop, but that would maybe limit some use cases where a reasonable amount of recursions is desired.

Reply:
That's a pretty big call out.
May need to update the readme to ensure that whoever implements this, that they do not update the triggering object in a way that would trigger another outbox message to be created for the same record or else they could run into this issue.

Reply:
It's actually fine if one outbox triggers another outbox for the same record... but it's not ok if it continues to happen indefinitely (EG there needs to be an exit condition).

Ya I was planning on adding something to the readme.